Classes/TceMain.php

```diff
@@ -35,8 +35,7 @@ public function processDatamap_afterDatabaseOperations(
         array $fieldArray,
         DataHandler $parentObject
     ) {
-
-        /**
```

Comment:
TYPO3 core use `/**` in most places.

Reply:
yes but `/**` is a doc comment while `/*` is a normal comment

Reply:
Thank you. That's the difference.